ph_rando/shuffler/macros.json

```diff
@@ -0,0 +1,3 @@
+{
+  "HitSwitch": "item Sword | item Boomerang"
```

Comment:
Obviously more items than just the sword or boomerang can hit switches, I wasn't sure off the top of my head so I just included these two.